reddit/mcrouter.sls

```diff
@@ -0,0 +1,25 @@
+{% from "reddit/map.jinja" import reddit with context %}
+
+create_mcrouter_configuration:
+  file.managed:
+    - name: /etc/mcrouter/global.conf
+    - makedirs: True
+    - contents: |
+        {{ reddit.mcrouter_config|json(indent=2)|indent(8) }}
+
+create_mcrouter_default_setting:
+  file.managed:
+    - name: /etc/default/mcrouter
```

Comment:
This file has no extension?

Reply:
Yes, that is correct. The convention in the /etc/defaults folder is to not have a file extension.